### DISCUSSION TITLE
Make all charges patternable

### DISCUSSION THREAD

**Discussion by Blipz** [Azgaar/Armoria#123] [Ideas]
It seems doable to make all charges patternable by changing:
```javascript
<pattern id="${p}" width="${size * .125}" height="${size * .125}">
```
to:
```javascript
<pattern id="${p}" width="${size * 25}" height="${size * 25}" patternUnits="userSpaceOnUse">
```
in `src/data/templates.js`.

Counterchanged charges will then have the same pattern size as the field below:
<img src="https://github.com/Azgaar/Armoria/assets/21015346/29a5faad-5c3e-4748-8f96-db676e90ee22" width="50%">

For this to be consistent across all charges, some work is needed on the SVG files to remove transforms where possible.

**Comment by Blipz** [comment]
Also, this opens the possibility to use a pattern on charges independently of counterchange.

**Comment by Azgaar** [comment]
We can try it, why not

**Comment by Blipz** [comment]
Merged here: 2d47a407ad3eb53aa7b851a31f4ce7cfad09dc93
